client/lcd/lcd_test.go

```diff
@@ -114,7 +114,7 @@ func TestCoinSend(t *testing.T) {
 
 	// query empty
 	res, body := Request(t, port, "GET", fmt.Sprintf("/auth/accounts/%s", someFakeAddr), nil)
-	require.Equal(t, http.StatusNoContent, res.StatusCode, body)
```

Comment:
we should add check for empty account (eg. `Account{}`)

Reply:
I added a test `TestAccountBalanceQuery` which covers checking if we are returning empty Account{}